### DISCUSSION TITLE
Truncate Modifier not working as expected?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#5984] [Troubleshooting] [answered]
Hi

I'm using `{{ desc_textarea | truncate:75:... }}`, unfortunately the ellipsis aren't being added to the end? I have tried a few variations to see if I could get the ellipsis to display but the word just cuts off, so it half works.

Anyone else spotted this?

```
Statamic 3.3.10 Pro
Laravel 9.5.1
PHP 8.0.8
simonhamp/statamic-heroicons 1.0.1
statamic/collaboration 0.4.0

Parser: runtime

```

**Comment by joshuablum** [comment, marked as answer]
Hey Stephen 👋

Please try the new syntax when using the `runtime` parser.

`{{ desc_textarea | truncate(75, '...') }}`

Does that solve your issue?

[Source #1](https://github.com/statamic/cms/issues/5833#issuecomment-1098174080)
[Source #2](https://statamic.dev/new-antlers-parser#modifiers)

**Reply by stephenmeehanuk** [reply]
Works a treat! Thanks for taking the time to reply, much appreciated.

**Reply by stephenmeehanuk** [reply]
Note. There's no mention of this in truncate docs...
https://statamic.dev/modifiers/truncate

Guess it'll be updated when the new parser is the default.

**Reply by jackmcdade** [reply]
Correct – the new syntax is only in the docs for the new parser.
